test/jasmine/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <script src="../../dist/js-reporters.js"></script>
+    <link rel="stylesheet" href="../../node_modules/jasmine-core/lib/jasmine-core/jasmine.css">
+    <script src="../../node_modules/jasmine-core/lib/jasmine-core/jasmine.js"></script>
+    <script src="../../node_modules/jasmine-core/lib/jasmine-core/jasmine-html.js"></script>
+    <script src="../../node_modules/jasmine-core/lib/jasmine-core/boot.js"></script>
+    <script src="tests.js"></script>
+    <script>
+
+        var runner = new JsReporters.JasmineAdapter();
+        //JsReporters.TapReporter.init(runner);
+        JsReporters.ConsoleReporter.init(runner);
```

Comment:
Maybe we can build some more infrastructure around this for testing purposes, where the output of both tap and console reporter are shown in an element on the page? That way we can just look at the rendered page, instead of having to check the console.

Somewhat related, we should verify that both reporters work when running inside a node process, outputting to the terminal.